config.py

```diff
@@ -131,10 +131,13 @@ class Config:
             "publish_benchmark_results": True,
         },
         "ec2-t3-xlarge-us-east-2": {
-            "info": "Supported benchmark langs: Python. Runs only benchmarks with cloud = True",
+            "info": "Supported benchmark langs: Python, R. Runs only benchmarks with cloud = True",
```

Comment:
```suggestion
            "info": "Supported benchmark langs: Python, R.",
```
Should this part now be removed (and replaced with something else)?